File: core/src/main/java/tubewarder/service/model/SendModel.java
package tubewarder.service.model;

import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlRootElement;
import java.util.List;
import java.util.Map;

@XmlRootElement
public class SendModel {
    @XmlElement(required = true)
    public String token;
    @XmlElement(required = true)
    public String template;
    @XmlElement(required = true)
    public String channel;
    @XmlElement(required = true)
    public AddressModel recipient;
    public Map<String, Object> model;
    public List<AttachmentModel> attachments;
    public String keyword;
    public String details;
}

File: static-next/app/model/send-model.ts
import { Address } from './address';
import { KeyValue } from './key-value';
import { Attachment } from './attachment';

export class SendModel {
    token: string;
    template: string;
    channel: string;
    recipient: Address;
    model: KeyValue[];
    attachments: Attachment[];
    keyword: string;
    details: string;
    echo: boolean;
}